addons/message_template/models.py

```diff
@@ -49,7 +49,7 @@ def send_group(self, group, sender=None, record=None, record_name=None):
         """
         Send a message to all users in a group `group`.
         """
-        group_obj = self.env.ref(group)
+        group_obj = self.sudo().env.ref(group)
         self.send(
             recipients=group_obj.users,
```

Comment:
i would put `sudo()` here, instead of above. Like this - `group_obj.sudo().users`. Besides limiting use of super user powers, this makes the reason we use `sudo()` much clearer to the reader - we use `sudo()` to access the full list of users.